legacyApp/app/activity/activity-routes.js

```diff
@@ -0,0 +1,39 @@
+module.exports = [
+    '$stateProvider',
+    '$urlMatcherFactoryProvider',
+function (
+    $stateProvider,
+    $urlMatcherFactoryProvider
+) {
+    $urlMatcherFactoryProvider.strictMode(false);
+
+    $stateProvider
+    .state({
+        name: 'activity',
+        url: '/activity',
+        controller: require('./activity.controller.js'),
+        template: require('./activity.html'),
+        /*resolve: {
+            loadDependencies: ['$ocLazyLoad', function($ocLazyLoad) {
+                //console.log('Testing!!!');
+                return import(/* webpackChunkName: "activity.module" *\/ './activity-module')
+                       .then(mod => $ocLazyLoad.load(mod.ACTIVITY_MODULE))
+                       .catch(err => {
+                           console.log('Hey, dosent work yet: ', err);
+                            //throw new Error('Hey, dosent work yet: ' + err)
+                       });
+                //return $ocLazyLoad.load('ushahidi.activity');
+            }]
+        }*/
+        lazyLoad: ($transition$) => {
```

Comment:
Whew! This part didn't work earlier because i wrote `lazyload` instead of `lazyLoad`. I still haven't been able to get the module split to happen yet, but at least it logs to console now.